src/app/@ansyn/map-facade/effects/map.effects.ts

```diff
@@ -101,7 +103,8 @@ export class MapEffects {
 		.map(({ mapId, toggleValue }): [CommunicatorEntity, boolean] => [this.communicatorsService.provide(mapId), toggleValue])
 		.filter(([comm, toggleValue]) => Boolean(comm))
 		.do(([comm, toggleValue]) => {
-			comm.setAutoImageProcessing(toggleValue);
+			const imageProccesingTool = comm.getPlugin<ImageProcessingPlugin>(ImageProcessingPlugin);
```

Comment:
Now  you can transfer this effects to `ImageProcessingPlugin`
please see northCalc plugin for example